spec/graphql/schema/input_object_spec.rb

```diff
@@ -108,4 +108,36 @@ class Schema < GraphQL::Schema
       assert_equal expected_info, res["data"]["inspectInput"]
     end
   end
+
+  describe "#to_h" do
+    module InputObjectToHTest
+      class TestInput1 < GraphQL::Schema::InputObject
+        graphql_name "TestInput1"
+        argument :d, Int, required: true
+        argument :e, Int, required: true
+      end
+
+      class TestInput2 < GraphQL::Schema::InputObject
+        graphql_name "TestInput2"
+        argument :a, Int, required: true
+        argument :b, Int, required: true
+        argument :c, TestInput1, as: :inputObject, required: true
+      end
+
+      TestInput1.to_graphql
+      TestInput2.to_graphql
+    end
+
+    it "returns a stringified, aliased, ruby keyword style hash" do
```

Comment:
Should this say "symbolized" instead of "stringified"?

Reply:
(I think besides that, this change is :+1:)

Reply:
fixed! thank you